types/combined-stream/index.d.ts

```diff
@@ -34,23 +43,18 @@ declare class CombinedStream extends Stream implements CombinedStream.Options {
     _emitError(error: Error): void;
 
     // events
-    on(event: "close" | "end" | "resume" | "pause", cb: () => void): this;
-    on(event: "error", cb: (err: Error) => void): this;
-    on(event: "data", cb: (data: any) => void): this;
-    once(event: "close" | "end" | "resume" | "pause", cb: () => void): this;
-    once(event: "error", cb: (err: Error) => void): this;
-    once(event: "data", cb: (data: any) => void): this;
-}
-
-declare namespace CombinedStream {
-    interface Options {
-        maxDataSize?: number;
-        pauseStreams?: boolean;
-    }
-
-    function create(options?: Options): CombinedStream;
-
-    function isStreamLike(stream: any): stream is Stream;
+    on(event: 'close' | 'end' | 'resume' | 'pause', cb: () => void): this;
+    on(event: 'error', cb: (err: Error) => void): this;
+    on(event: 'data', cb: (data: any) => void): this;
+    once(event: 'close' | 'end' | 'resume' | 'pause', cb: () => void): this;
+    once(event: 'error', cb: (err: Error) => void): this;
+    once(event: 'data', cb: (data: any) => void): this;
+
+    create(options?: Options): CombinedStream;
+    isStreamLike(stream: any): stream is Stream;
```

Comment:
These two members are not defined on the prototype so they should only be static as below.

Reply:
Done.